.github/workflows/helper_scripts/request_contributor_review.py

```diff
@@ -0,0 +1,142 @@
+import argparse
+import requests
+import os
+import sys
+import json
+from pathlib import Path
+
+REPO_OWNER = "demisto"
+REPO_NAME = "content"
+PACKS_FOLDER = "Packs"
+CONTENT_REPO_FULL_PATH = os.environ.get('GITHUB_WORKSPACE') or os.path.abspath(
+    os.path.join(__file__, '../../../..'))
+PACKS_FULL_PATH = os.path.join(CONTENT_REPO_FULL_PATH, PACKS_FOLDER)
+PACK_METADATA = "pack_metadata.json"
+XSOAR_SUPPORT = "xsoar"
+PACK_METADATA_GITHUB_USER_FIELD = "githubUser"
+
+
+def check_if_user_exists(github_user, github_token=None, verify_ssl=True):
+    user_endpoint = f"https://api.github.com/users/{github_user}"
+    headers = {'Authorization': 'Bearer ' + github_token} if github_token else {}
+
+    response = requests.get(user_endpoint, headers=headers, verify=verify_ssl)
+
+    if response.status_code not in [200, 201]:
+        print(f"Failed in pulling user {github_user} data")
+        sys.exit(1)
+
+    github_user_info = response.json()
+
+    if 'id' in github_user_info:
+        return True
+    else:
+        return False
+
+
+def get_pr_author(pr_number, github_token, verify_ssl):
+    pr_endpoint = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/pulls/{pr_number}"
+    headers = {"Authorization": "Bearer " + github_token} if github_token else {}
+
+    response = requests.get(pr_endpoint, headers=headers, verify=verify_ssl)
+
+    if response.status_code not in [200, 201]:
+        print(f"Failed in pulling PR {pr_number} data")
+        sys.exit(1)
+
+    pr_info = response.json()
+
+    return pr_info.get('user', {}).get('login', '').lower()
+
+
+def get_pr_modified_packs(pr_number, github_token, verify_ssl):
+    pr_endpoint = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/pulls/{pr_number}/files"
+
+    headers = {'Authorization': 'Bearer ' + github_token} if github_token else {}
+
+    response = requests.get(pr_endpoint, headers=headers, verify=verify_ssl)
+
+    if response.status_code not in [200, 201]:
+        print(f"Failed in pulling PR {pr_number} data")
+        sys.exit(1)
+
+    pr_changed_data = response.json()
+    pr_files = [f.get('filename') for f in pr_changed_data]
+    modified_packs = {Path(p).parts[1] for p in pr_files if p.startswith(PACKS_FOLDER) and len(Path(p).parts) > 1}
+
+    return modified_packs
+
+
+def request_review_from_user(reviewers, pr_number, github_token=None, verify_ssl=True):
```

Comment:
not sure we can ask external users for a review (outside demisto organization).
please test and switch to comment if not possible. (if you don't have a personal user - use mine: shaiy310)

Reply:
Changed to comment instead